src/lib/components/Swap/Swap.fixture.tsx

```diff
@@ -25,6 +25,12 @@ function Fixture() {
     }
   }, [color, setColor])
 
+  const [convenienceFee] = useValue('convenienceFee', { defaultValue: 100 })
+  const [convenienceFeeRecipient] = useSelect('convenienceFeeRecipient', {
+    options: ['', '0x0dA5533d5a9aA08c1792Ef2B6a7444E149cCB0AD', '0xE6abE059E5e929fd17bef158902E73f0FEaCD68c'],
```

Comment:
mega nit: but mayeb save as constants so we know what these options are? idk maybe not important